conf.yaml

```diff
@@ -190,6 +189,12 @@ repository:
                 -
                     action: assign_tag
                     action_value: 'popularity/low'
+                 -
+                    action: remove_tag
+                    action_value: 'popularity/medium
```

Comment:
https://github.com/martenson/p4/blob/remove-old-poptags/process.py#L180 is how I checked for presence of a tag. 

You can probably basically copy+paste that.